### ISSUE TITLE
Page exclusion from docs

### ISSUE THREAD

**Issue by matteofrana** [shuding/nextra#59]
Hi Shu,
I'd like to have the docs root in /pages/docs, but have a content site in the /pages root folder.
So I'd like to have a way to exclude pages from the docs navigation or set the documentation root folder.

What do you think?
Matteo

**Comment by davehorton** [comment]
I am interested in this feature as well...is there currently a way to accomplish this?

**Comment by shuding** [comment]
Yeah this will be an important feature to support (setting project root).

**Comment by davehorton** [comment]
Is this a change in nextra, or in next itself?  I would be happy to take a shot at making the change if you could point me in the right direction...

**Comment by kitajchuk** [comment]
It seems like you might be able to add an extra config option or use the `path` config option to determine which files/directories to generate the `pageMap`.

https://github.com/shuding/nextra/blob/core/packages/nextra-theme-docs/src/index.js#L174

**Comment by talentlessguy** [comment]
does anyone know how to set a docs root right now?

**Comment by mattvb91** [comment]
has anyone managed to solve this?

**Comment by CarlPhilippe** [comment]
You can deploy your application as a sub-path in nextjs. Learn more: https://nextjs.org/docs/api-reference/next.config.js/basepath

There should be a way to set up the rewrites in next.config, I don't know how to.

**Comment by NasirAbdullahSyed** [comment]
Need this!
